### ISSUE TITLE
Pick up wrong timezone

### ISSUE THREAD

**Issue by ntung** [williamtroup/Calendar.js#244]
When I update an event:
![image](https://github.com/user-attachments/assets/b3323a1c-0b60-43e7-971d-c5d4dea5811e)

The plugin picks up wrong timezone.
```
Clicked me - Tung!
{"from":"2024-08-20T17:30:00.000Z","to":"2024-08-20T18:15:00.000Z","title":"Tuan Nguyen, 4 people, M: 01223666606","description":"Email: nvntung@outlook.com, Phone: 01223666606, for 4 people with the message Chi Hai dat ban.","location":"Thanh Binh Restaurant","group":"Booking Group","id":"eb6cb10b-4d6c-4b5d-3859-7f53b9e2913c","created":"2024-08-03T19:09:47.312Z"}
```

**Comment by williamtroup** [comment]
It takes the times from the form.  Timezone issues are apparent, and I will hopefully have an option for this in v3.0.0+.

**Comment by ntung** [comment]
Thank you for the swift reply!
Should I figure out a trick to fix the issue on our end?

**Comment by williamtroup** [comment]
Yes, for now, it does pick the values correctly from the form, so you will get what they have selected.  You will need to convert it on your end for storage.

**Comment by ntung** [comment]
As you see, it is 1 hour later than the values on the form. I think it is because of BST and GMT. Right?

**Comment by williamtroup** [comment]
Yes, that will be the issue.  I will be adding better support for this moving forward into v3.0.0.
